client/css/style.css

```diff
@@ -915,6 +915,7 @@ kbd {
 #chat .text {
 	flex: 1 1 auto;
 	overflow: hidden;
+	white-space: pre-wrap;
```

Comment:
You'll need to specify `white-space: normal` for .toggle, because this messes that up.